cli/consumer_command.go

```diff
@@ -1555,11 +1555,7 @@ func (c *consumerCmd) prepareConfig() (cfg *api.ConsumerConfig, err error) {
 		}
 
 		if cfg.Durable != "" && c.consumer != "" && cfg.Durable != c.consumer {
-			if c.consumer != "" {
-				cfg.Durable = c.consumer
-			} else {
-				return cfg, fmt.Errorf("durable consumer name in %s does not match CLI consumer name %s", c.inputFile, c.consumer)
-			}
+			cfg.Durable = c.consumer
```

Comment:
we would still need to return the error in a else though right for when both are set but different

Reply:
actually never mind, I am being dumb.